.travis.yml

```diff
@@ -35,16 +61,30 @@ jobs:
       # Docs and lint python versions must be synced with those in tox.ini
     - python: '3.8'
       env: TOX_ENV=docs
+    - python: '3.8'
+      arch: arm64
+      env: TOX_ENV=docs
     - python: '3.8'
       env: TOX_ENV=lint
   allow_failures:
     # Currently failing due to https://foss.heptapod.net/pypy/cffi/issues/458
     - python: nightly
 
 install:
+    - |
```

Comment:
I think this should go in the `script` phase instead of the `install` phase. That puts it after the diagnostic prints (`python -VV`, `curl-config`, etc) and means there's just one `if/else` on `BUILD_WHEEL` instead of an `if -n` one place and an `if -z` in another.

Reply:
Correct, Thanks for the suggestion. Modified the same.